### PR TITLE
Api update

### PR DESCRIPTION
**Pull request by victorfonsec4**
Updates the client to use the new api with alerts and trigalerts

### CODE REVIEW

File: lib/zeus/api_client/alerts_interface.rb [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+      end
+    end
+
+    # enable alerts
+    # @param [Array[Integer]] alert_id_list list of alert ids to be enabled
+    # @return [Zeus::APIClient::Result]
+
+    def enable_alerts(alert_id_array)
+      params = {id: alert_id_array}
+      p params
```
Comment (travelist) [inline lib/zeus/api_client/alerts_interface.rb]: Remove

``` ruby
p params
```

File: lib/zeus/api_client/alerts_interface.rb [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+      end
+    end
+
+    # disable alerts
+    # @param [Array[Integer]] alert_id_list list of alert ids to be disabled
+    # @return [Zeus::APIClient::Result]
+
+    def disable_alerts(alert_id_array)
+      params = {id: alert_id_array}
+      p params
```
Comment (travelist) [inline lib/zeus/api_client/alerts_interface.rb]:
``` ruby
p params
```

File: lib/zeus/api_client/trigalerts_interface.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        Result.new(response)
+      rescue => e
+        Result.new(e.response)
+      end
+    end
+
+    # get triggered alerts in last 24 hours
+    # @return [Zeus::APIClient::Result]
+
+    def get_triggered_alerts_last_24_hours()
```
Comment (travelist) [inline lib/zeus/api_client/trigalerts_interface.rb:23]: Actually, `begin` statement is redundant for these small functions.
So we can write like following (actually this is my fault):

``` ruby
    def get_triggered_alerts_last_24_hours
      response = get("/trigalerts/#{@access_token}/last24")
      Result.new(response)
    rescue => e
      Result.new(e.response)
    end
```

We dont need to re-write like this, but just I let you know ;)

**Comment by travelist** [comment]
@victorfonsec4 Thank you for the pull request. I cheked the entire code and looks good to me:) 👍

**Review by travelist — APPROVED** [review]

**Comment by victorfonsec4** [comment]
Thanks!

**Pull request merged**